### ISSUE TITLE
Fails to match when using ${PWD##*/} in glob pattern

### ISSUE THREAD

**Issue by MhmRhm** [matepek/vscode-catch2-test-adapter#374]
I am doing out of source builds. My build directory name begins with my project name which is my current directory name. I want to use my current directory name in glob pattern. Running following linux command can find the test:

```bash
~/repos/MyProject$ find ../${PWD##*/}-build-*/test/**/*test
../MyProject-build-linux-default-debug/test/infrastructure/infrastructure_test
```

`../${PWD##*/}-build-*/test/**/*test` does not work in `testMate.cpp.test.executables`. I am using gtest.

**Comment by matepek** [comment]
"current directory" doesn't make sense in the context of vscode, do you mean workspace directory?

**Comment by MhmRhm** [comment]
Yes. Thanks for mentioning. Am I totally off here? I don't know is that a feature that needs to be added or that's a bug. Or achieving such think is possible in the domain of vscode.

**Comment by matepek** [comment]
Usually vscode opens directories. The root folder can be referred as `${workspaceFolder}`. I think this is what you need.
For more variable check [this](https://github.com/matepek/vscode-catch2-test-adapter/blob/master/documents/configuration/test.advancedExecutables.md#variables).

**Comment by MhmRhm** [comment]
This works `../${workspaceName}-build-*/test/**/*test`. Thanks very much.
